### ISSUE TITLE
[Bug]: Pendant: Links outside post content aren't underlined.

### ISSUE THREAD

**Issue by scruffian** [Automattic/themes#7505]
### Quick summary

In Pendant (and probably other themes), links outside the Post Content have underline turned off:

https://github.com/Automattic/themes/blob/ca887cd3f72c1b755c0b5b6d2fe2c85d7597411c/pendant/theme.json#L378

If I edit my home page to add a link to it, there is no way to make that link be underlined, without using a keyboard shortcut (see https://github.com/WordPress/gutenberg/issues/56321).

We should reconsider this approach as it makes it difficult to for websites to be accessible.

cc @beafialho 

### Steps to reproduce

1. Switch to Pendant
2. Edit the Blog Home template
3. Add a new paragraph with a link
4. Notice that the link is not underlined

### What you expected to happen

There should be a way to underline this link.

### What actually happened

It's impossible unless I use a keyboard shortcut.

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

None

**Comment by liviopv** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Replicated
- Tested on Atomic – Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO

https://github.com/Automattic/themes/assets/14153300/07fde16b-799f-4db8-a264-0416a0b43b2f


📌 ACTIONS
- Triaged

📌 Message to Author
@scruffian it's possible to make the link underlined by selecting it from the block tool bar as shown towards the end of the video above.

**Comment by scruffian** [comment]
Thanks, that is a workaround, but I don't think we should expect users to have to do this for every link.
